src/main/resources/com/typesafe/sbt/packager/archetypes/scripts/ash-template

```diff
@@ -1,105 +1,155 @@
 #!/bin/sh
```

Comment:
I really like the cleaned up version of this scripts. This is by far more readable and predictable